tensorflow/contrib/distributions/python/kernel_tests/deterministic_test.py

```diff
@@ -40,11 +41,11 @@ def testShape(self):
 
   def testInvalidTolRaises(self):
     loc = rng.rand(2, 3, 4).astype(np.float32)
-    deterministic = deterministic_lib.Deterministic(
-        loc, atol=-1, validate_args=True)
-    with self.cached_session():
-      with self.assertRaisesOpError("Condition x >= 0"):
-        deterministic.prob(0.).eval()
+    with self.assertRaisesRegexp(errors.InvalidArgumentError,
+                                 "Condition x >= 0"):
+      deterministic = deterministic_lib.Deterministic(
```

Comment:
please fix the lint error "Unused variable 'deterministic' [unused-variable]"

Reply:
Changed `deterministic` to `_`